gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-debug/src/main/java/io/gravitee/gateway/debug/reactor/handler/context/steps/DebugStep.java

```diff
@@ -110,16 +108,18 @@ public void stop() {
     public void error(Throwable ex) {
         this.stop();
         this.status = DebugStepStatus.ERROR;
-        this.diffMap.put(DIFF_KEY_ERROR_MESSAGE, ex.getMessage());
+        this.error = new DebugStepError();
+        this.error.setMessage(ex.getMessage());
```

Comment:
Maybe we could have prefer a dedicated constructor ? `new DebugStepError(ex.getMessage)` ?